### DISCUSSION TITLE
Is a full read-only mode possible?

### DISCUSSION THREAD

**Discussion by DeltaTangoLima** [photoprism/photoprism#2184] [Q&A]
In lieu of multiple user support, I was wondering if it's possible to setup a second instance that reflects the exact same content and database as my primary instance, but is read-only?  An instance that doesn't even have the ability to change albums, etc.

My use case is to have a primary instance that's only accessible via my private network, then a full read-only instance that we can use, even publicly via my nginx reverse proxy, to view our photos and videos.

Both our phones sync their camera rolls via Syncthing when on the private network, so all I want to do is use the main instance to curate/organise the content, and the read-only instance to view it.

If I had a second user in MariaDB that could only _read_ the database, and used that with $PHOTOPRISM_READONLY in a second instance, would that do the trick?

**Comment by graciousgrey** [comment]
PHOTOPRISM_READONLY only refers to your originals directory see https://docs.photoprism.app/getting-started/config-options/.
If i is set, albums could still be created or settings changed.

Adding users with read-only access will be part of multi user support.

**Reply by DeltaTangoLima** [reply]
No, I get that. I'm just wondering if I could use a read only user in MariaDB to bridge the gap until work on #98 is complete.
